src/components/input-dropdown/input-dropdown.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="ux-select-container">
+    <button #button type="button" class="form-control"
+            [uxMenuTriggerFor]="menu">
+        <ng-template #defaultDisplayContent>{{selected ? (selected | json) : '-'}}</ng-template>
+        <ng-container [ngTemplateOutlet]="displayContentRef || defaultDisplayContent"></ng-container>
+    </button>
+    <div class="ux-select-icons">
+        <ux-icon name="close"
+                 class="ux-select-icon ux-select-clear-icon"
+                 *ngIf="allowNull && selected"
+                 (click)="resetValue($event)">
+        </ux-icon>
+        <ux-icon name="chevron-down"
+                 class="ux-select-icon ux-select-chevron-icon"
+                 (click)="menuTrigger.toggleMenu(); $event.stopPropagation()">
+        </ux-icon>
+    </div>
+</div>
+
+<ux-menu #menu menuClass="select-menu"
+         (opened)="filterInputElement.nativeElement.focus()">
+
+    <div [style.max-height]="maxHeightString"
+         [style.width.px]="button.offsetWidth">
+
+        <div *ngIf="!hideFilter"
+             class="filter-container">
+
+            <input #filterInput
+                    type="text"
+                    [placeholder]="placeholder"
+                    class="form-control"
+                    [(ngModel)]="filterText"
+                    (input)="filterChange.emit(filterText)"
+                    (click)="$event.stopPropagation()" />
+
+            <button type="button"
+                    class="btn btn-flat btn-icon-only filter-button"
```

Comment:
There is no `btn-icon-only` class. `btn-icon` perhaps? https://uxaspects.github.io/UXAspects/#/css/buttons#icon-buttons-documentation-component

Reply:
Removed.